README.md

```diff
@@ -1,7 +1,7 @@
 # cltdevs meetup manifesto
 
 ## Mission statement
-We grow Charlotte's tech community by bringing people together.
+We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating individual talents and organizations, and discover and build the missing elements of a thriving tech community.
```

Comment:
Also "individuals and organizations" is mentioned twice. An alternative phrasing could be something like this:

We are committed to the Charlotte tech community's growth and sustainability by fostering relationships between individuals and organizations, cultivating talents and teams, and discovering the missing elements of a thriving tech community so that we can improve them.